src-vue/src/plugins/store.ts

```diff
@@ -361,11 +366,19 @@ export const store = createStore<FlightCoreStore>({
  * It invokes all Rust methods that are needed to initialize UI.
  */
 async function _initializeApp(state: any) {
-    // Enable dev mode directly if application is in debug mode
-    if (await invoke("is_debug_mode")) {
-        store.commit('toggleDeveloperMode', true);
-    } else {
-        // Disable context menu in release build.
+    // Display dev view if dev mode was previously enabled or if debug mode is active.
+    // Menu style is only modified on debug mode.
```

Comment:
Don't forget to update the comment ;)

Reply:
Addressed in 7236093389910eda2bb2cbcb9700b35a0ea14b7a